R/data.R

```diff
@@ -497,11 +498,9 @@ data_load_request <- function(path_data, hash = NULL, quiet = FALSE) {
   path_req <- data_path_request(path_data)
   if (is.null(hash)) {
     keys <- data_admin_list_requests(I(path_data))
-    nk <- length(keys)
-    workflow_log(
-      quiet,
-      ngettext(nk, "There is 1 request for access",
-               sprintf("There are %d requests for access", nk)))
+    n <- length(keys)
+    what <- ngettext(n, "request", "requests")
+    workflow_log(quiet, sprintf("There is %d %s request for access", n, what))
```

Comment:
Tiny thing but there is a dupe of request here - you're going to get `There is 1 request request for access` or `There is 2 requests request for access`